cmd/tailscale/cli/update.go

```diff
@@ -44,8 +44,8 @@ var updateCmd = &ffcli.Command{
 		fs := newFlagSet("update")
 		fs.BoolVar(&updateArgs.yes, "yes", false, "update without interactive prompts")
 		fs.BoolVar(&updateArgs.dryRun, "dry-run", false, "print what update would do without doing it, or prompts")
-		fs.StringVar(&updateArgs.track, "track", "", `which track to check for updates: "stable" or "unstable" (dev); empty means same as current`)
-		fs.StringVar(&updateArgs.version, "version", "", `explicit version to update/downgrade to`)
+		fs.StringVar(&updateArgs.track, "track", "", `which track to check for updates: "stable" or "unstable" (dev); empty means same as current; ignored on Arch-based distros`)
```

Comment:
I wonder if we can gate printing this additional caveat on GOOS=linux? It won't mean anything to Windows or macOS people, and might even confuse them.

Reply:
Added a condition for these flags to only register on non-Arch hosts :+1: